Bottom: ecd51a1eb836b55fc01e88b5629c0d27e4167d68
Top:    2447047049893adaec9f5ec656ae4eaa0eae0e32
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 20:55:17 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
